src/Components/App/App.js

```diff
@@ -12,6 +12,9 @@ class App extends Component {
       this.state = {
         movies: [],
         movieSelected: false,
+        matchingMovies: null,
+        isRendered: false,
```

Comment:
We may need an empty array up here.  Then we may not need the matching movies propery.